source/extensions/http/header_formatters/preserve_case/preserve_case_formatter.h

```diff
@@ -14,17 +16,26 @@ namespace PreserveCase {
 class PreserveCaseHeaderFormatter : public Envoy::Http::StatefulHeaderKeyFormatter {
 public:
   // Envoy::Http::StatefulHeaderKeyFormatter
-  PreserveCaseHeaderFormatter(const bool forward_reason_phrase);
+  PreserveCaseHeaderFormatter(
+      const bool forward_reason_phrase,
+      const envoy::extensions::http::header_formatters::preserve_case::v3::
+          PreserveCaseFormatterConfig::FormatterTypeOnEnvoyHeaders formatter_type_on_envoy_headers);
 
   std::string format(absl::string_view key) const override;
   void processKey(absl::string_view key) override;
   void setReasonPhrase(absl::string_view reason_phrase) override;
   absl::string_view getReasonPhrase() const override;
+  Envoy::Http::HeaderKeyFormatterOptConstRef formatterOnEnvoyHeaders() const;
 
 private:
   StringUtil::CaseUnorderedSet original_header_keys_;
   bool forward_reason_phrase_{false};
   std::string reason_phrase_;
+  envoy::extensions::http::header_formatters::preserve_case::v3::PreserveCaseFormatterConfig::
```

Comment:
This doesn't need to be initialized as it is initialized in the constructor. You can also make it const.

Reply:
done